src/util.coffee

```diff
@@ -32,3 +32,25 @@
                                       node?.tag is 'tag:yaml.org,2002:map'
 
 @NON_PRINTABLE = /[^\x09\x0A\x0D\x20-\x7E\x85\xA0-\uD7FF\uE000-\uFFFD]/
+
+@stringifyJSONCircularDepth = (json, depth = 1000) ->
```

Comment:
Hi @dmartinezg!

Actually the util function default is 1000 but the use in raml.coffee default is 500. I thought that value was more realistic (Ref: https://github.com/raml-org/raml-js-parser/pull/180/files#diff-fe0283a18f459b13627ffad0f22b8eb5R161)

I have also signed the contributor agreement.

Thanks.